tracing-subscriber/src/fmt/format/pretty.rs

```diff
@@ -174,13 +174,12 @@ where
                 if let Some(name) = thread.name() {
                     write!(writer, "{}", name)?;
                     if self.display_thread_id {
-                        write!(writer, " ({:?})", thread.id())?;
```

Comment:
hmm, this changes the formatting slightly because we'll no longer have parentheses around thread IDs when thread names are also enabled. i'm not sure if this is particularly important, though.

Reply:
good point, I didnt notice those extra brackets.
However looking at it I think this is an improvement as the thread id is already displayed as `ThreadId(1)` so having `(ThreadId(2))` is a bit weird.

Reply:
sure, i don't have a strong opinion on whether the parentheses are good or not, although they seemed more _grammatically_ correct here. not a big deal either way though.

Reply:
normally, i'd try to reserve formatting changes for a major version bump, but this seems reasonable to me.